### ISSUE TITLE
GUI based viewer?

### ISSUE THREAD

**Issue by medcl** [jprante/elasticsearch-skywalker#1]
raw json is hard,can you pls provide a simple gui,thanks.

**Comment by jprante** [comment]
Yes, a GUI is needed. But there is also Elasticsearch Head http://mobz.github.com/elasticsearch-head/ . Ben Birch is interested in a skywalker extensio, but this depends on a REST endpoint for a plugin list. Elasticsearch will soon have an extension to list all plugins installed, in the nodes status info.

**Comment by medcl** [comment]
that's great, very glad to hear that~

---

------------------ Original ------------------
From: "J?rg Prante"; 
Date: 2012Äê6ÔÂ21ÈÕ(ÐÇÆÚËÄ) ÍíÉÏ8:51
To: "Medcl"; 
Subject: Re: [elasticsearch-skywalker] GUI based viewer? (#1)

Yes, a GUI is needed. But there is also Elasticsearch Head http://mobz.github.com/elasticsearch-head/ . Ben Birch is interested in a skywalker extensio, but this depends on a REST endpoint for a plugin list. Elasticsearch will soon have an extension to list all plugins installed, in the nodes status info.

---
